apps/src/blockly/addons/cdoTrashcan.js

```diff
@@ -77,12 +77,17 @@ export default class CdoTrashcan extends GoogleBlockly.DeleteArea {
       },
       bodyClipPath
     );
+    const SPRITE = {
```

Comment:
i'm not sure how strongly i feel about this in this case, but we usually move constants (as in actual constants like this, not just `const` usages) to the top of the file for easy discoverability

Reply:
Thanks Maddie - I was weighing the pros/cons of doing it that way, or like this, where the definition shows up right before it is needed for the first time. No strong preference, so I'm happy to move it up before merging!